ksmt-core/src/main/kotlin/org/ksmt/solver/KSolver.kt

```diff
@@ -57,6 +63,8 @@ interface KSolver : AutoCloseable {
      * @see check
      * @see unsatCore
      * */
+    @Suppress("INAPPLICABLE_JVM_NAME")
```

Comment:
Is it correct? Shouldn't it be something like "JvmOverload" instead? I just don't get why is it inapplicable and how this jvmName will help with default arguments

Reply:
`JvmName` is not supported for interfaces: https://youtrack.jetbrains.com/issue/KT-31420. 

The Kotlin compiler applies mangling to functions with default arguments in some cases. JvmName allows us to disable mangling and therefore makes the functions accessible from Java.

Reply:
It would be nice to use JvmOverload, but it is also unsupported for interfaces and unlike JvmName it can't be suppressed: https://youtrack.jetbrains.com/issue/KT-36102